.travis.yml

```diff
@@ -3,21 +3,25 @@
 # are not executed on Aarch64.
 arch: arm64
 services:
-    - docker
+  - docker
 jobs:
-    include:
+  include:
     - name: "Integration tests using Mbed Crypto provider"
       env:
-      - DOCKER_IMAGE_NAME=mbed-crypto-provider
-      - DOCKER_IMAGE_PATH=tests/per_provider/provider_cfg/mbed-crypto
-      # Stress test fails on Travis CI, see #116
-      - SCRIPT="tests/ci.sh --no-stress-test mbed-crypto"
+        - DOCKER_IMAGE_NAME=mbed-crypto-provider
+        - DOCKER_IMAGE_PATH=tests/per_provider/provider_cfg/mbed-crypto
+        # Stress test fails on Travis CI, see #116
+        - SCRIPT="tests/ci.sh --no-stress-test mbed-crypto"
```

Comment:
I think we can remove the `--no-stress-test` and put them in the `allow_failures` if they start failing consistently